test/AutoDiff/SIL/Parse/sildeclref.sil

```diff
@@ -1,5 +1,5 @@
 // RUN: %target-sil-opt %s -module-name=sildeclref_parse | %target-sil-opt -module-name=sildeclref_parse | %FileCheck %s
-
+// XFAIL: *
```

Comment:
??

Reply:
oh no, thanks for spotting those -- seems I left a few more in than I should have